components/core/tools/scripts/lib_install/check-cmake-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Exit on any error
+set -e
+
+# Error on undefined variable
+set -u
+
+# Get the installed cmake version string
+cmake_version=$(cmake -E capabilities | jq ".version.string")
+cmake_major_version=$(cmake -E capabilities | jq ".version.major")
+
+# Check if version is 4.0 or higher
+# shellcheck disable=SC2071
+if ! [[ "$cmake_major_version" < "4" ]]; then
```

Comment:
We can compare directly against `cmake_version`, right?

```suggestion
if ! [[ "$cmake_version" < "4" ]]; then
```

Reply:
fixed in the newest commit.
Need to pass `--raw-output` flag to `jq` to remove the extra quotes around cmake version string